test/libsolidity/gasTests/dispatch_small.sol

```diff
@@ -2,15 +2,15 @@ contract Small {
     uint public a;
     uint[] public b;
     function f1(uint x) public returns (uint) { a = x; b[uint8(msg.data[0])] = x; }
-    function () external payable {}
+    fallback () external payable {}
 }
 // ----
 // creation:
-//   codeDepositCost: 83600
+//   codeDepositCost: 84600
```

Comment:
Why does it spend more gas?

Reply:
`calldata` check?

Reply:
The optimized version below doesn't change in gas, so I guess it's fine.